### PR TITLE
6 files uploaded

### PR DESCRIPTION
**Pull request by HadeerAIA**
Please check these files

### CODE REVIEW

**Review by ayushrana182 — CHANGES_REQUESTED** [review]
there's a spell-check issue in on of the file, could you correct that please?

**Comment by HadeerAIA** [comment]
> there's a spell-check issue in on of the file, could you correct that please?

In which file please?

**Comment by ayushrana182** [comment]
doc/source/continuous-probability-distribution.md:205:274: "binominal" is a misspelling of "binomial"

**Comment by HadeerAIA** [comment]
> doc/source/continuous-probability-distribution.md:205:274: "binominal" is a misspelling of "binomial"

It's not in the 6 files included in this pull request 
the files that uploaded was "Staccatone , Optophonic_Piano, Synthetic_Tone, Theremin, Electrophon, Hugoniot_Organ"  
"continuous-probability-distribution.md" Not one of them

**Comment by HadeerAIA** [comment]
> It's just a few second's change, I requested because you are the only one pushing changes now

Oh ok, I was thinking just about the 6 files right now, But ok I will check the spelling and push it again.

File: doc/source/hugoniot_organ.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+"With this scheme the various types of wave forms for different timbres may be placed in radial sectors on a disk; another disk carrying the scanning slits in circular tracks rotates before this wave-form disk. A source of light and photocell complete the translating arrangements. Each slit track scans its corresponding wave cycle at a speed corresponding to one pitch of an
+approximate tempered scale. Thus, one wave and one slit track serve for each tone frequency of the tempered scale. Naturally the lowest pitch tracks are nearest the center and the highest are nearest the circumference of the scanning disk.
+
+![hugoniot_organ](_static/images/hugoniot_organ/hugoniot_organ.png)
```
[the remark this answers is not in the document]
Comment (HadeerAIA) [reply doc/source/hugoniot_organ.md]: Done

File: doc/source/hugoniot_organ.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+similar scanning slits, equispaced on a continuous track.
+
+The slit spacing is precisely equal to the wave-form lengths,
+so that this wave form is repeated at the scanning frequency; i.e., the number of slits passing it per second. The same method was disclosed as early as 1921 by the French inventor Hugoniot, who described an electrical musical instrument of this type in his patent’"
+
+[A description of Hugoniot’s photo-electrical sound generation method from ‘Electronic Music and Instruments’ Institute of Radio Engineers, 1936]
+
+Hugoniot’s died in 1927 before he could develop his ideas any further than prototypes yet he left behind a legacy of innovation that influenced a new generation of French pioneering instrument designers including Pierre Toulon and Givelet & Coupleaux.
+
+Next topic: Theremin
```
[the remark this answers is not in the document]
Comment (HadeerAIA) [reply doc/source/hugoniot_organ.md:22]: I have reordered the files according to the next topic mentioned in the PDF files sent, you could review it again and tell me if there is anything left.

**Pull request merged**